x-pack/platform/plugins/shared/lens/public/datasources/form_based/operations/definitions/column_types.ts

```diff
@@ -6,18 +6,18 @@
  */
 
 import type { Query } from '@kbn/es-query';
-import type { Operation } from '../../../../types';
 import type { TimeScaleUnit } from '../../../../../common/expressions';
 import type { OperationType } from '.';
 
-export interface BaseIndexPatternColumn extends Operation {
+export interface BaseIndexPatternColumn {
   // Private
   operationType: string;
-  customLabel?: boolean;
```

Comment:
rather than setting this bool and then also providing the label always, just set the label when its custom, else leave it unset